Cargo.toml

```diff
@@ -26,20 +26,19 @@ cosmwasm_1_4 = ["cosmwasm_1_3", "cosmwasm-std/cosmwasm_1_4"]
 multitest_api_1_0 = []
 
 [dependencies]
-cw-utils = "1.0"
-cw-storage-plus = "1.1"
-cosmwasm-std = { version = "1.4", features = ["staking", "stargate"] }
-itertools = "0.11"
-schemars = "0.8"
-serde = { version = "1.0", default-features = false, features = ["derive"] }
-prost = "0.11"
-anyhow = "1.0"
-thiserror = "1.0"
-derivative = "2"
-sha2 = "0.10"
+cw-utils = "1.0.1"
+cw-storage-plus = "1.1.0"
+cosmwasm-std = { version = "1.4.0", features = ["staking", "stargate"] }
+itertools = "0.11.0"
+schemars = "0.8.13"
+serde = { version = "1.0.188", default-features = false, features = ["derive"] }
+prost = "0.12.0"
+anyhow = "1.0.75"
+thiserror = "1.0.48"
+derivative = "2.2.0"
+sha2 = "0.10.7"
 
 [dev-dependencies]
-# We don't use the following dependency directly,
-# we tighten the version so that builds with `-Zminimal-versions` work.
-serde_json = "1.0.105"
-ecdsa = "=0.16.7"
\ No newline at end of file
+# We don't use these dependencies directly,
+# we tighten versions that builds with `-Zminimal-versions` work.
+ecdsa = "0.16.8"
```

Comment:
Check out https://editorconfig.org/ for avoiding such missing linebreaks

Reply:
Fixed in #68.